packages/std/src/errors/std_error.rs

```diff
@@ -477,10 +477,12 @@ impl From<DivideByZeroError> for StdError {
 pub type StdResult<T> = core::result::Result<T, StdError>;
 
 #[derive(Error, Debug, PartialEq, Eq)]
+#[non_exhaustive]
 pub enum OverflowOperation {
     Add,
     Sub,
     Mul,
+    Neg,
```

Comment:
Would it be possible to move all Neg implementations to 2.0? Then we can avoid this breaking change here. This still ships most of the value of this PR quickly and users can still do x*-1 as a workaround.